client/css/style.css

```diff
@@ -1,24 +1,25 @@
-*{
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-}
+
 
 .app p {
   color: blue;
 }
 
-html,body{
-  width: 100%;
-  height: 100%;
+.reviewsMain {
   font: normal 12px/18px 'Helvetica Neue', Helvetica, Arial, sans-serif;
-  background-color: #f2f2f2;
   color: #414141;
   -webkit-font-smoothing: antialiased;
   text-rendering: optimizeLegibility;
 }
 
-h1,h2,h3,h4,h5,h6,p{
+
+
+.reviewsMain h1,
+.reviewsMain h2,
+.reviewsMain h3,
+.reviewsMain h4,
+.reviewsMain h5,
+.reviewsMain h6,
+.reviewsMain p{
```

Comment:
any default tags you want to override needs to still be scoped otherwise all our h1s and ps and etc will be overritten .. this can be done by prefacing it with that main wpr class thing